frontend/lib/norent/homepage.tsx

```diff
@@ -248,11 +269,13 @@ export const NorentHomePage: React.FC<{}> = () => {
                   </div>
                   <div>
                     <p className="title is-size-4 jf-alt-title-font">
-                      Legal Protections
+                      <Trans>Legal Protections</Trans>
```

Comment:
Oops I just realized these are also in Title Case?!  wat

Reply:
Hmmm, I'm wondering if I should actually check in with Tahnee about these— there might be reasoning as to why certain titles are title case and others are sentence case and I don't wanna overwrite if there was an intention there.

Reply:
See https://github.com/JustFixNYC/tenants2/pull/1405#issuecomment-627415847 below